functions/number-in-between/1.0/index.js

```diff
@@ -0,0 +1,22 @@
+const numberInBetween = async ({
+  collection,
+  leftProperty,
+  rightProperty,
+  value,
+  returnProperty,
+}) => {
+  const { data } = collection;
+
+  const result = data.filter(
```

Comment:
What should happen once the collection has a lenght of 1 or 0? Maybe catch that exception and throw an error